packages/react-reconciler/src/ReactFiberNewContext.new.js

```diff
@@ -157,10 +157,6 @@ export function scheduleWorkOnParentPath(
       !isSubsetOfLanes(alternate.childLanes, renderLanes)
     ) {
       alternate.childLanes = mergeLanes(alternate.childLanes, renderLanes);
-    } else {
-      // Neither alternate was updated, which means the rest of the
-      // ancestor path already has sufficient priority.
```

Comment:
This comment may not be correct in cases where `Context.Consumer` is the ancestor of `Suspense`.

(But with this PR, it goes to all the ancestors, so please let me know if you have a better way to do it)

Reply:
This does seem pretty aggressive, right? It seems like this is an important optimization here. I think you need to figure out why the assumption in this code is not true, and how we can either make it true, or change the check in a way that doesn't regress all other cases.

Reply:
Thank you for your comment, I'll figure the root cause out.

※Current situation; it [bailoutOnAlreadyFinishedWork](https://github.com/okmttdhr/react/blob/fb36d983ee5684ab0994eccc7d670a6a822ae35d/packages/react-reconciler/src/ReactFiberBeginWork.new.js#L3337) on a function component that wraps `Context.Consumer` before it get to `Context.Consumer` ([This childLanes](https://github.com/okmttdhr/react/blob/fb36d983ee5684ab0994eccc7d670a6a822ae35d/packages/react-reconciler/src/ReactFiberBeginWork.new.js#L3113) would be `Context.Consumer`'s parent)

Reply:
@gaearon 
Changed it to schedule only needed work.
https://github.com/facebook/react/pull/21414/commits/ed589927ba84cb36342918b26f7062be3d927ef9